homeassistant/components/anova_cooker/translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Delete this file, it will be filled based on strings.json and translations can be added.